rust/libnewsboat/src/filterparser.rs

```diff
@@ -15,3 +21,134 @@ enum Expression {
         value: Value
     },
 }
+
+named!(operators<CompleteStr, CompleteStr>,
+    do_parse!(
+        tag: alt!(
+            tag!("==") |
+            tag!("!=") |
+            tag!("=~") |
+            tag!("=") |
+            tag!("!~") |
+            tag!("<") |
+            tag!(">") |
+            tag!("<=") |
+            tag!(">=") |
+            tag!("between") |
+            tag!("#") |
+            tag!("!#")
+        ) >>
+        (tag)
+    )
+);
+
+fn quoted_string(input: CompleteStr) -> IResult<CompleteStr, Value>{
+    do_parse!(input,
+        tag!("\"")
+        >> value: escaped_transform!(
+                    is_not!("\\\""),
```

Comment:
It won't match `"\"` because it's missing another quote as a tag, and it panics (another TODO: Proper error handling).

Demo program for the filterparser to try it out until there are proper tests here: https://gist.github.com/tsipinakis/41ec505a6b1997daa18284698df360c0.